resources-service/src/resource-jobs/client-sqs.ts

```diff
@@ -66,8 +79,11 @@ export const publishToResourcesJob = async (params: ResourcesSearchIndexPayload,
 
 export const publishSearchIndexJob = (accountSid: string, resource: ReferrableResource) => {
   return publishToResourcesJob({
+    params: {
       accountSid,
       jobType: ResourcesJobType.SEARCH_INDEX,
       document: resource,
+    },
+    messageGroupId: resource.id,
```

Comment:
Nit: Should probably include the account SID in the group ID just for logical correctness - it probably doesn't have much potential impact resources from different accounts with ID clashes into the same group but at least it makes the intention clear if we include it